pkg/ctl/gitops/apply.go

```diff
@@ -153,7 +141,9 @@ func doApplyGitops(cmd *cmdutils.Cmd, opts options) error {
 	if err != nil {
 		return err
 	}
-	usersRepoDir := filepath.Join(opts.outputPath, usersRepoName)
+	dir, err := ioutil.TempDir("", usersRepoName)
+	logger.Debug("Directory %s will be used to clone the configuration repository and install the profile", dir)
```

Comment:
I am thinking if it would be better to have it as an Info message instead. I think I would be wondering where it did all the operations.

Reply:
I specifically didn't want to log it at `Info` level because I thought this is only useful to _debug_ potential issues, and because Git operations are already logged at `Info` level, and do mention the temporary directory, e.g.:
```
Cloning into '/var/folders/24/d3mml6bn20nftpt91cfldq1h0000gn/T/my-gitops-repo979274957/my-gitops-repo'...
```
versus, before (see #1336), e.g.:
```
Cloning into '/Users/marc/dev/src/github.com/weaveworks/eksctl/_test/my-gitops-repo'...
```